example/package.json

```diff
@@ -39,6 +39,7 @@
     "react-native-gesture-handler": "^2.9.0",
     "react-native-keychain": "^8.1.1",
     "react-native-markdown-display": "^6.1.6",
+    "react-native-notifications": "^4.3.3",
```

Comment:
This is an unrelated change to the PR's focus, but it's required to get the example app back up & running.